pkg/policy/api/rule.go

```diff
@@ -438,19 +438,18 @@ type PortRuleHTTP struct {
 type PortRuleKafka struct {
 	// Role is a case-insensitive string and describes a group of API keys
 	// necessary to perform certain higher level Kafka operations such as "produce"
-	// or "consume". An APIGroup automatically expands into all APIKeys required
+	// or "consume". A Role automatically expands into all APIKeys required
 	// to perform the specified higher level operation.
 	//
 	// The following values are supported:
 	//  - "produce": Allow producing to the topics specified in the rule
 	//  - "consume": Allow consuming from the topics specified in the rule
 	//
-	// This field is incompatible with the APIKey field, either APIKey or Role
-	// may be specified.
-	//
-	// If omitted or empty, the field has no effect and the logic of the APIKey
-	// field applies.
+	// This field is incompatible with the APIKey field, i.e APIKey and Role
+	// cannot both be specified in the same rule.
 	//
+	// If omitted or empty, all keys are allowed.
```

Comment:
This needs to be adjusted.

Reply:
Changed to `If omitted or empty,
  and if Role is not specified, then all keys are allowed.`